### ISSUE TITLE
Distroted audio in The Dynastic Hero

### ISSUE THREAD

**Issue by Wesleyrpg** [Mazamars312/openfpga-pcengine-cd#40]
Just wanted to give you a heads up, that the audio in The Dynastic Hero is sounding kinda funky and distorted!

Thanks for all the work!

**Comment by Mazamars312** [comment]
Yep, there is a lot of work I need to do to fixup in this core. Will be working on this soon
